lua/nvim-treesitter-textobjects/move.lua

```diff
@@ -36,11 +36,15 @@ end
 local M = {}
 
 ---@param opts TSTextObjects.MoveOpts
-local function move(opts)
-  local query_group = opts.query_group or "textobjects"
-  local query_strings = type(opts.query_strings) == "string" and { opts.query_strings } or opts.query_strings
+---@param query_strings string[]|string
+---@param query_group? string
+local function move(opts, query_strings, query_group)
+  query_group = query_group or "textobjects"
+  if type(query_strings) == "string" then
+    query_strings = { query_strings }
+  end
 
-  local winid = opts.winid or api.nvim_get_current_win()
+  local winid = api.nvim_get_current_win()
```

Comment:
why was `opts.winid` removed?

Reply:
The exposed API never passes `winid`, so it will always be nil and `get_current_win()`.

Reply:
I thought it was used somewhere, but honestly everyone uses textobjects and movements within the current window. So I guess it's not a problem to remove them or we should add winid everywhere to make it consistent.

Reply:
In our current exposed API, the second parameter is already optional. If we add a third parameter as an optional one, when we want to pass the third parameter, we would at least have to pass `nil` as the second parameter. So, in my opinion, unless it's particularly useful, it's better not to add it.

Reply:
I think that's why we had it in the `opts`, no?

Reply:
In most APIs with optional parameters in neovim, yes. However the API in this plugin does not have `opts`, it has two fixed parameters. The function with `opts` parameter is internally used.